### PR TITLE
fix: set primary node on new profile

### PR DESCRIPTION
**Pull request by nicole-obrien**
# Description of change

Issue was that staking wasn't loading when the profile was new, until the user went to the networking settings. This was because when the profile is first created it doesn't set any of the official nodes in the official network config to the primary node. I added some code to select a random one as the primary node.

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Fix (a change which fixes an issue)

## How the change has been tested

Tested manually on MacOS by recovering a profile using a 24 word mnemonic.

## Change checklist

Add an `x` to the boxes that are relevant to your changes, and delete any items that are not.

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that my latest changes pass CI tests
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: packages/shared/lib/network.ts [hunk 1]
```diff
@@ -29,13 +29,19 @@ export const CHRYSALIS_DEVNET_BECH32_HRP = 'atoi'
...
-export const getOfficialNetworkConfig = (type: NetworkType): NetworkConfig => ({
-    network: getOfficialNetwork(type),
-    nodes: getOfficialNodes(type),
-    automaticNodeSelection: true,
-    includeOfficialNodes: true,
-    localPow: true,
-})
+export const getOfficialNetworkConfig = (type: NetworkType): NetworkConfig => {
+    const officialNodes = getOfficialNodes(type)
+    const randIdx = Math.floor(Math.random() * officialNodes.length)
```
Comment (maxwellmattryan) [inline packages/shared/lib/network.ts:34]: There is a [`pick`](https://github.com/iotaledger/firefly/blob/feat/staking/packages/shared/lib/utils.ts#L351) function in `lib/utils.ts` that will randomly select the element of an array - we can de-dup some code a bit and substitute it here.

Comment (nicole-obrien) [reply packages/shared/lib/network.ts:34]: @maxwellmattryan that returns the specific element of the array, where I want to modify a specific element in an array and keep the rest of the array too.

Although there is this exact code further down that I can extract into its own function.

Comment (nicole-obrien) [reply packages/shared/lib/network.ts]: Made it a little more DRY 😊

**Review by maxwellmattryan — APPROVED** [review]
Very noice 👍

**Pull request merged**